rasa/cli/arguments/run.py

```diff
@@ -32,6 +32,19 @@ def add_port_argument(
     )
 
 
+def add_interface_argument(
+    parser: Union[argparse.ArgumentParser, argparse._ArgumentGroup]
+) -> None:
+    """Binds the RASA process to a network interface."""
+    parser.add_argument(
+        "-i",
+        "--interface",
```

Comment:
I feel like `--host` might be more idiomatic.

Reply:
Tried using that when I started working on it, but found an issue with the short argument: `-h`, which is generally used for help. Then I thought about adding `-i` and `--host`, but that looked even more confusing so eventually settled on `-i`. Happy to change it though.

Reply:
Good point - let's keep it as `-i`